src/types.ts

```diff
@@ -43,6 +43,16 @@ export type TokenMintParams = {
   amount: string | number;
 };
 
+/**
+ * The parameters to pass to the NFT transfer functions.
+ *
+ * @beta
+ */
+export type TokenParams = {
+  toAddress: WalletAddress;
+  amount: string | number;
+};
+
```

Comment:
let's merge this with tokenMintparams like you did but keep it `to` instead of `toAddress`